tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -265,13 +265,169 @@ Ramdump received successfully..!
 DumpParser Script provides two interfaces: CUI and GUI
 
 ### DumpParser using CUI
+
+#### To get call stack using RAM dump
 1. Run Ramdump Parser Script
 ```
 cd $TIZENRT_BASEDIR/tools/dump_tool/
 python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
 2. See the Output
 
+#### To display Debug Symbols/Crash point using assert logs
+1. Copy crash logs
+    First copy the crash logs to a file named log_file in tools/dump_tool/`<log_file>`
+2. Run Script in tools/dump_tool path
+    $ python3 ramdumpParser.py -e `<Elf path>` -t `<Log file path>`
+
+    ex)
+    $ python3 ramdumpParser.py -e ../../build/output/bin/wifi_dbg -t ./log_file
+
+Example Call Stack Output for App crash is as follows:
+```
+*************************************************************
+dump_file         : None
+log_file          : .log_file
+elf_file  : ../../../output/wifi_dbg
+*************************************************************
+
+Number of applicaions : 2
+App1 is : micom
+App2 is : wifi
+
+----------------------------------------------------------
+App Crash point is as follows:
+App name : wifi
+0x00001492
+main
+/root/tizenrt/loadable_apps/loadable_sample/wifiapp/wifiapp.c:85
+
+0x00003367
+binary_manager_send_request
+/root/tizenrt/framework/src/binary_manager/binary_manager_interface.c:97
```

Comment:
What is second info?

Reply:
This shows the addr2line output based on R14 & R15 values

Reply:
1. As far as I know, pc register has current operation +4 (thumb) or +8 (arm) so that it does not show the location which causes the problem. Is there any need to print this?
2. No one knows what they mean. Please write they show info of R14 and R15.

Reply:
1. Yes, in case of no up_registerdump() we are getting approximate crash point using pc value only
2) Added appropriate description